packages/crypto/README.md

```diff
@@ -1,7 +1,38 @@
 # @turnkey/crypto
 
-This package consolidates some common cryptographic utilities used across our applications, particularly primitives related to keys, encryption, and decryption. For react-native you will need to polyfill our random byte generation by importing react-native-get-random-values: https://www.npmjs.com/package/react-native-get-random-values
+This package consolidates some common cryptographic utilities used across our applications, particularly primitives related to keys, encryption, and decryption in a pure JS implementation. For react-native you will need to polyfill our random byte generation by importing react-native-get-random-values: https://www.npmjs.com/package/react-native-get-random-values
 
 ```
 import 'react-native-get-random-values'
 ```
+
+Example usage (Hpke E2E):
+
+```
+const senderKeyPair = generateP256KeyPair();
+const receiverKeyPair = generateP256KeyPair();
+
+const receiverPublicKeyUncompressed = uncompressRawPublicKey(
+  uint8ArrayFromHexString(receiverKeyPair.publicKey),
+);
+
+const plaintext = "Hello, this is a secure message!";
+const encryptedData = hpkeEncrypt({
+  plainText: plaintext,
```

Comment:
I think this needs to be updated now that you take a uint8array instead of a string